core/src/test/java/org/apache/iceberg/inmemory/TestInMemoryFileIO.java

```diff
@@ -27,11 +27,11 @@
 import org.junit.jupiter.api.Test;
 
 public class TestInMemoryFileIO {
-  String location = "s3://foo/bar.txt";
 
   @Test
   public void testBasicEndToEnd() throws IOException {
     InMemoryFileIO fileIO = new InMemoryFileIO();
+    String location = "s3://foo/bar.txt";
```

Comment:
can we just use random file names with UUIDs in this case, instead of have one dedicated name for each test?